tests/_openmp_test_helper/parallel_sum.pyx

```diff
@@ -1,13 +1,17 @@
 import cython
+cimport openmp
 from cython.parallel import prange
 from libc.stdlib cimport malloc, free
 
+
 def parallel_sum(int N):
     cdef double Ysum = 0
-    cdef int i
+    cdef int i, num_threads
     cdef double* X = <double *>malloc(N*cython.sizeof(double))
 
     for i in prange(N, nogil=True):
+        num_threads = openmp.omp_get_num_threads()
```

Comment:
Why do you put this call inside the `prange` loop? It should yield the same result in the main thread, right?

Reply:
No, this command gives the effective number of thread so out of orange, it returns 1.